src/tests/ftest/interoperability/updown_grade.py

```diff
@@ -0,0 +1,353 @@
+                 (3)./launch.py upgrade_downgrade -ts boro-[..] -tc boro-[..]
+        :avocado: tags=manual
+        :avocado: tags=interop
+        :avocado: tags=upgrade_downgrade
+        """
+        # (1)Setup
+        hosts_client = self.hostlist_clients
+        hosts_server = self.hostlist_servers
+        all_hosts = include_local_host(hosts_server)
+        upgd_rpms = self.params.get("upgrade_rpms", '/run/interop/*')
+        downgd_rpms = self.params.get("downgrade_rpms", '/run/interop/*')
+        num_attributes = self.params.get("num_attributes", '/run/attrtests/*')
+        mount_dir = self.params.get("mount_dir", '/run/dfuse/*')
+        self.log.info("(1)==Show rpm, dmg and daos versions on all hosts.")
+        self.show_daos_version(all_hosts, hosts_client)
+
+        # (2)Create pool containers and attributes
+        self.log.info("(2)==Create pool containers and attributes.")
+        self.add_pool(connect=False)
+        pool_id = self.pool.identifier
+        self.add_container(self.pool)
+        self.container.open()
+        self.daos_cmd = self.get_daos_command()
+        attr_dict = self.create_data_set(num_attributes)
+
+        try:
+            self.container.container.set_attr(data=attr_dict)
+            data = self.daos_cmd.container_list_attrs(
+                pool=self.pool.uuid,
+                cont=self.container.uuid,
+                verbose=False)
+            self.verify_list_attr(attr_dict, data['response'])
+
+            data = self.daos_cmd.container_list_attrs(
+                pool=self.pool.uuid,
+                cont=self.container.uuid,
+                verbose=True)
+            self.verify_get_attr(attr_dict, data['response'])
+        except DaosApiError as excep:
+            print(excep)
+            print(traceback.format_exc())
+            self.fail("Test was expected to pass but it failed.\n")
+        self.container.close()
+        self.pool.disconnect()
+
+        # IOR before upgrade
+        self.log.info("(2.1)==Setup and run IOR.")
+        result = run_pcmd(hosts_client, "mkdir -p {}".format(mount_dir))
+        ior_timeout = self.params.get("ior_timeout", '/run/ior/*')
+        iorflags_write = self.params.get("write_flg", '/run/ior/iorflags/')
+        testfile = os.path.join(mount_dir, "testfile")
+        testfile_sav = os.path.join(mount_dir, "testfile_sav")
+        testfile_sav2 = os.path.join(mount_dir, "testfile_sav2")
+        symlink_testfile = os.path.join(mount_dir, "symlink_testfile")
+        self.ior_cmd.flags.update(iorflags_write)
+        self.run_ior_with_pool(
+            timeout=ior_timeout, create_pool=False, create_cont=False, stop_dfuse=False)
+        result = run_pcmd(hosts_client, "cd {}".format(mount_dir))
+        result = run_pcmd(hosts_client, "ls -l {}".format(testfile))
+        result = run_pcmd(hosts_client, "cp {0} {1}".format(testfile, testfile_sav))
+        self.check_result(result)
```

Comment:
```suggestion
        result = run_pcmd(hosts_client, "cd {}".format(mount_dir))
        self.check_result(result)
        result = run_pcmd(hosts_client, "ls -l {}".format(testfile))
        self.check_result(result)
        result = run_pcmd(hosts_client, "cp {0} {1}".format(testfile, testfile_sav))
        self.check_result(result)
```
Don't we want to call check_result after running each command?